cmd/federation-controller/main.go

```diff
@@ -134,21 +135,20 @@ func main() {
 	}()
 
 	var fdsClient *adsc.ADSC
-
 	remote := cfg.MeshPeers.Remote
-
 	if len(remote.Addresses) > 0 {
-		remoteName := remote.Name
```

Comment:
@bartoszmajsak if it comes to these little formatting changes:
1. I don't see value in multiplying variables if the new ones are not notably shorter.
2. I like compact pieces of code within a function, like this:
```
// I do something
var a
var b
condition that uses above variables

// I do something else
var c
var d
func1(c)
func2(d)
```

Reply:
Fair enough

Reply:
I think it's some leftover from me porting multi-peer WIP stuff, I also didn't like it when I noticed it now. So we are on the same page here.

Reply:
I think what you described is called "cuddling" :)